static/app/utils/marked.tsx

```diff
@@ -62,7 +62,7 @@ marked.setOptions({
     }
 
     loadPrismLanguage(lang, {
-      onLoad: () => callback?.(null, Prism.highlight(code, Prism.languages[lang], lang)),
+      onLoad: () => callback?.(null!, Prism.highlight(code, Prism.languages[lang], lang)),
```

Comment:
What is the error message without using `!`?

Reply:
> Argument of type 'null' is not assignable to parameter of type 'Error'.ts(2345)

Prototype of callback is
```
(parameter) callback: ((error: Error, code?: string | undefined) => void) | undefined
```

I used this answer https://stackoverflow.com/a/66994737 but without deep understanding.

Reply:
instead of null!, you can pass undefined

Reply:
> Argument of type 'undefined' is not assignable to parameter of type 'Error'.ts(2345)